notebooks/official/pipelines/custom_model_training_and_batch_prediction.ipynb

```diff
@@ -76,27 +76,27 @@
       "source": [
         "### Objective\n",
         "\n",
-        "In this tutorial, you learn to use `Vertex AI Pipelines` and `Google Cloud Pipeline Components` to build a custom model.\n",
+        "In this tutorial, you learn to use Vertex AI Pipelines and Google Cloud Pipeline components to build a custom model.\n",
```

Comment:
"Components" should start with an uppercase "C" to be consistent with the GCPC documentation and the Vertex AI documentation. 

Reference: https://source.corp.google.com/piper///depot/google3/googledata/devsite/site-cloud/en/vertex-ai/_local_variables.html;l=207?q=%22setvar%20gcpc_name%22&sq=package:piper%20file:%2F%2Fdepot%2Fgoogle3%20-file:google3%2Fexperimental

Reply:
This is something we'll need to discuss. At present, give its status, it should be lower case.